hedera-node/src/main/java/com/hedera/services/utils/MiscUtils.java

```diff
@@ -337,6 +338,18 @@ public static String readableTransferList(TransferList accountAmounts) {
 				.toString();
 	}
 
+	public static String readableTokenTransferList(TokenTransferList tokenTransferList) {
```

Comment:
Lets rename it to `readableNftTransferList` since we are only parsing and setting nft transfers

Reply:
Confirmed!